api/src/utils/database-type-helper.ts

```
@@ -0,0 +1,156 @@
+  },
+};
+
+/**
+ * Helper for numeric column types
+ */
+export const NumberColumnType = {
+  /**
+   * Type for integer columns
+   * @param defaultValue Default integer value
+   */
+  integer: (defaultValue: number = 0): ColumnOptions => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)